make/e2e-setup.mk

```diff
@@ -334,7 +334,7 @@ e2e-setup-pebble: load-bin/downloaded/containers/$(CRI_ARCH)/pebble.tar bin/scra
 
 bin/downloaded/containers/$(CRI_ARCH)/samplewebhook/samplewebhook: make/config/samplewebhook/sample/main.go $(DEPENDS_ON_GO)
 	@mkdir -p $(dir $@)
-	GOOS=linux GOARCH=$(CRI_ARCH) $(GOBUILD) -o $@ $(GOFLAGS) make/config/samplewebhook/sample/main.go
+	GOOS=linux GOARCH=$(CRI_ARCH) $(GOBUILD) -o $@ make/config/samplewebhook/sample/main.go
```

Comment:
question: why did this change? it seems reasonable to pass GOFLAGS here?

Reply:
Fair enough!

I think maybe my ideal would be that even if the `-X` directives are ignored, we still don't pass un-used directives around if it's avoidable; but I don't see any reason to spend a lot of time on it here, it's an easy enough improvement to make in the future!